CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# How to contribute
+
+Operator Marketplace is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, instructions on how to set up a dev environment, and other resources to help get contributions into operator-marketplace.
+
+# Email and Chat
```

Comment:
There's no email listed here under the Email header. Additionally, that link is to the operator-framework google group which we don't control and afaik even look at. We don't have a public email list set up either, so is this header necessary? Maybe we should set up something and add it later?

Reply:
That is a good point. Main way to reach out should be issues I guess.